app/components/RegionDropdown/RegionDropdown.module.scss

```diff
@@ -1,75 +1,59 @@
 @import "../../styles/colors";
+@import "../../styles/vars";
+@import "../../styles/typography";
+
+// Focus-ring (move somewhere more global later)
+@mixin focus-ring {
+  box-shadow: 0 0 0 2px var(--control-focus-ring);
+}
 
 .dropdown-container {
   position: relative;
   cursor: pointer;
-  width: 16.25rem;
+  width: calc($space-3xl * 7);
   color: var(--text-positive);
 
-  //Animation is currently off
+  // Animation is currently off — I don't know if this is what we want anyways because the dropdown could be closed but still in focus?
```

Comment:
Yeah most likely only have it animated when the dropdown is open instead of on focus

Reply:
IDK how to do that though :(

Reply:
Ah, I don't think its possible with this version. Can do it in v2 ;)